assets/js/frontend/components/review/index.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import ReadMore from '../read-more';
+
+function getReviewClasses( isLoading ) {
```

Comment:
Should we use `classnames` lib instead? It's pretty lightweight.

Reply:
Agree it's quite small and we might consider using it. My only objection is that currently all external modules we import generate a new request. I guess we could update webpack's config so small modules are bundled instead of loaded separately, would that something we want to do?